packages/fx-core/src/common/azure-hosting/azureHosting.ts

```diff
@@ -72,13 +71,31 @@ export abstract class AzureHosting {
     } as ResourceTemplate;
   }
 
+  static replaceHolder(module: string, pluginId: string): string {
```

Comment:
We can leverage the method name of HandleBars or Mustache for this method name. Or use "renderTemplate()"?